geomstats/geometry/product_manifold.py

```diff
@@ -1,4 +1,8 @@
+<<<<<<< HEAD
```

Comment:
You've got leftover merge conflict labels in here.

You'll have to merge the code here somehow, then remove those lines:
<<<<<<<<<<<<< HEAD, etc

Reply:
+1